template_helpers.js

```diff
@@ -388,18 +388,28 @@
         return DDG.getImageProxyURL(url);
     });
 
+    /**
+     * @function imageProxyThumbnail
+     *
+     * Request thumbnail images in default size + 25%
+     */
+
+    Handlebars.registerHelper("imageProxyThumbnail", function(url) {
+        return DDG.getImageProxyURL(url, false, false, 204);
```

Comment:
Hard coding 204 in here and wrapping it as a generic function seems odd. If it's a generic helper function, it should probably be a variable. And if 204 is hard coded, maybe it should just be a function inside the image IA.

Another approach could be added a second size argument to the regular `imageProxy` helper? Maybe that would be more reusable?